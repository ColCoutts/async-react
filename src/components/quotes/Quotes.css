.Quotes {
  display: flex;
  justify-content: space-around;
  flex-wrap: wrap;
  background-color: #889fa3;

  li {
    width: 20vw;
    margin: 2rem;
    padding: 2rem;

    background-color: #dff0f3;
  }

  section {
    margin: auto;
  }

  img {
    width: 10rem;
    margin: auto;
  }
}
