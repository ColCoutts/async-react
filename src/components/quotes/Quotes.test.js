import React from 'react';
import { shallow } from 'enzyme';
import Quotes from './Quotes';

describe('Quotes compenent', () => {
  it('renders a quote', () => {
    const quotes = [
      {
        character: 'Fry',
        quote: 'wuat',
        image: './image'
      },
      {
        character: 'Fry',
        quote: 'wuat',
        image: './image'
      },
      {
        character: 'Fry',
        quote: 'wuat',
        image: './image'
      }
    ];

    const wrapper = shallow(<Quotes quotes={quotes} />);
    expect(wrapper).toMatchSnapshot();
  });
});
