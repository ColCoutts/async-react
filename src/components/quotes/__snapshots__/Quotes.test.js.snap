// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotes compenent renders a quote 1`] = `
<ul
  className="Quotes"
>
  <li
    key="0"
  >
    <Quote
      quote={
        Object {
          "character": "Fry",
          "image": "./image",
          "quote": "wuat",
        }
      }
    />
  </li>
  <li
    key="1"
  >
    <Quote
      quote={
        Object {
          "character": "Fry",
          "image": "./image",
          "quote": "wuat",
        }
      }
    />
  </li>
  <li
    key="2"
  >
    <Quote
      quote={
        Object {
          "character": "Fry",
          "image": "./image",
          "quote": "wuat",
        }
      }
    />
  </li>
</ul>
`;
